bindings/python/test.py

```diff
@@ -75,6 +76,18 @@ def test_torrent_status(self):
 		st2 = self.h.status()
 		self.assertEqual(st2, st)
 
+	def test_status_last_uploaded_dowloaded(self):
+		# we want to check at seconds precision but can't control session
+		# time, wait for next full second to prevent second increment
+		time.sleep(1 - datetime.datetime.now().microsecond / 1000000.0)
+
+		sessionStart = datetime.datetime.now().replace(microsecond=0)
```

Comment:
And what should I do if I got a 1 sec increment between befor and after start? I would use after start, but what to do if I got a second increment before and after add torrent? I just don't know the time. But at least session start takes less then 1 sec its reliable.
Or I could calculate a time diff and write assert(diff < seconds(2))